lib/galaxy/managers/users.py

```diff
@@ -489,7 +489,9 @@ def send_reset_email(self, trans, payload={}, **kwd):
                 return "Failed to produce password reset token. User not found."
 
     def get_reset_token(self, trans, email):
-        reset_user = trans.sa_session.query(self.app.model.User).filter(func.lower(self.app.model.User.table.c.email) == email.lower()).first()
+        reset_user = trans.sa_session.query(self.app.model.User).filter(self.app.model.User.table.c.email == email.lower()).first()
```

Comment:
@mvdbeek Just noticed while merging forward: should this have been:

```python
        reset_user = trans.sa_session.query(self.app.model.User).filter(self.app.model.User.table.c.email == email).first()
```

(i.e. without the `.lower()` on `email`) ?

Reply:
Yes

Reply:
https://github.com/galaxyproject/galaxy/pull/9446